configs/setups/awicm3/awicm3.yaml

```diff
@@ -706,9 +706,9 @@ oasis3mct:
                 true:
                         lag: "${time_step}"
                         # If ini_restart_dir is not specified in the runscript and, therefore,
-                        # it is not a branched off experiment add the /fesom.nproc/ to the
+                        # it is not a branched off experiment adds the /fesom.nproc/ to the
                         # sources when the run_number is 1
-                        load_restart_from_pool: "$(( '${input_dir}' == '${ini_restart_dir}' and ${general.run_number} == 1))"
+                        load_restart_from_pool: "$(( os.path.realpath('${input_dir}') == os.path.realpath('${ini_restart_dir}') and ${general.run_number} == 1 ))"
```

Comment:
Since I have memory. Is it a bad idea? Potentially. Would removing this capability kill many features from AWICM3 and make us have to implement even more syntax into the yaml? for sure

Reply:
Not thumbs down to how you did it, thumbs down to...just....mmmmpf. I probably should have told Dirk this is dangerous from the beginning before it was so established.

Reply:
Is it? All of our users can just edit the backend anyway if they like.

Reply:
Well, maintenance cost is the factor here: If users NEED to change the backend by hand, then they can do whatever they want. Most people are afraid of that -- it's the backend, we have often enough said: you look at this only if you know exactly what you are doing -- so it's unlikely to happen. If users could just change one little yaml and then (given clever yaml skills) erase their entire folder, and then come to ask us why its all gone, that is a less happy scenario (again: this will never actually happen)

Reply:
If the user want to delete their work dir we can't stop them. Life finds a way ;-)

Reply:
Honestly, I see the danger on it, but I cannot live without it. I wouldn't expect that anyone writes a long script here, mostly because it needs to be a one-liner and you cannot create variables in the backend outside of the config via this method. That is IMO safe enough. On the other hand, this allows us to do complex boolean comparisons which is excellent for `choose_` blocks, without complicating choose blocks or having to invent an if syntax. In summary I love it.

And one funny note, `$(( ))` syntax of ESM-Tools was implemented by you guys thinking to allow math operations. I think you did not imagine that someone will find that anything you put inside the parenthesis would run within an `eval()`. Sorry, is partly my fault for finding it ;)